app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -37,7 +37,7 @@ class MainActivity : BaseActivity(), SyncManager.SyncListener {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        if (savedInstanceState != null && savedInstanceState?.containsKey("tag")) {
+        if (savedInstanceState != null && savedInstanceState.containsKey("tag")) {
```

Comment:
I think in this case it's an slight optimization, since we've already established that it's not null with "savedInstanceState != null". Not really necessary to change, but it doesn't need to be there.